js_modules/dagit/packages/core/src/execute/LaunchButton.tsx

```diff
@@ -63,7 +64,11 @@ export const LaunchButton = ({config, runCount}: LaunchButtonProps) => {
     runCount,
     disabled: config.disabled,
   });
+  const logButtonClick = useTelemetryAction("Clicked LaunchPad Button");
```

Comment:
rather than doing arbitrary strings, what do you think about setting up some sort of action enum, maybe some value string, along with some metadata blob?

something like this:
```
const logLaunch = useTelemetryAction(
    UserAction.LAUNCH_RUN,
    'launchpad',
    {opSelection: null, hasCustomTags: true}
);
```

Reply:
Definitely in agreement regarding this structure, I think we don't want to log the actual opSelection value though. Maybe just check if they specified an op selection at all i guess